sdk/batch/batch/test/batchClient.spec.ts

```diff
@@ -303,8 +308,8 @@ describe("Batch Service", function () {
         await client.pool.add(pool);
         assert.fail("Expected error to be thrown");
       } catch (error) {
-        assert.equal(error.statusCode, 403);
-        assert.equal(error.body.code, "Forbidden");
+        assert.equal(error.statusCode, 400);
```

Comment:
Do we need this test still? It seems like it was previously asserting that VNets didn't work with cloud service configuration set. If we do want a test to make sure we're handling `InvalidPropertyValue` responses correctly, that makes sense to me, but then can we make it something really obviously wrong (like a string literal "invalid_valid")?

Reply:
Oh I was wondering why the error code changed. Previous was due to VNet doesn't work with Cloud Service, right now it's saying the subnetId is invalid.

Reply:
let me remove this test